### PR TITLE
Build sky on Fuchsia

### PR DESCRIPTION
**Pull request by pylaligand**

### CODE REVIEW

File: BUILD.gn [hunk 1]
```diff
@@ -5,21 +5,15 @@
...
-      "//flutter/runtime",
-      "//flutter/snapshotter",
-      "//flutter/snapshotter($host_toolchain)",
-    ]
-  } else {
-    deps = [
-      "//flutter/sky",
-    ]
-  }
+  deps = [
```
Comment (pylaligand) [inline BUILD.gn]: Not sure this is what you meant with the TODO I removed.

Comment (pylaligand) [inline BUILD.gn]: Well the build is failing so I guess this is not what you meant :)

**Comment by abarth** [comment]
LGTM

... but Travis is mad

**Comment by chinmaygarde** [comment]
@pylaligand `is_fuchsia_host` is not in our buildroot.  You can either add it there and roll the buildroot or use `defined()` and check if the flag is available.

**Comment by pylaligand** [comment]
I added it in a separate change: https://github.com/flutter/buildroot/pull/36

**Comment by pylaligand** [comment]
I do need to update the reference to the buildroot somewhere in flutter/engine, don't I? Build is still broken after I pushed the changed above.

**Comment by chinmaygarde** [comment]
Reference can be updated [here](https://github.com/flutter/engine/blob/327d122de9eaf3739996610a3630408bdcc52103/DEPS#L49)

**Comment by pylaligand** [comment]
Yeah my latest commit has it, and Travis is now happy.

**Pull request merged**